cart/questions.py

```diff
@@ -98,8 +98,13 @@ def get_questions(self):
         return root
 
     def write_to_file(self, file):
-        tree = ET.ElementTree(self.get_questions())
-        tree.write(file, encoding="unicode")
+        from xml.dom.minidom import parseString
```

Comment:
There is already a write_xml function with a prettify option in util.py. Please use that.